rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,153 @@
+### DCP PR:
+***Leave this blank until the RFC is approved then the Author(s) must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc\#\](https://github.com/HumanCellAtlas/dcp-community/pull/\<PR\#\>)`
+
+# Technical Support SOP
+
+## Summary
+
+This process RFC describes how the DCP handles technical support requests through the use of the Zendesk application and other processes.
+
+## Author(s)
+
+`[Kevin Osborn](mailto:kosborn2@ucsc.edu)`
+
+`[Jodi Hirschman](mailto:jhirsch@broadinstitute.org)`
+
+`[Matthew Spier](mailto:mspeir@ucsc.edu)`
+
+`[Mallory Freeberg](mailto:mfreeberg@ebi.ac.uk)`
+
+## Shepherd
+
+***Leave this blank. This role is assigned by DCP PM to guide the Author(s) through the RFC process.*
+
+## Motivation
+
+Beta 2 testing will begin soon and we’ve decided to collect feedback
```

Comment:
This feels like a very short term motivation. Surely we need to define a policy which will work beyond beta2 and review and refine it as demands change rather than defining a very short term policy

Reply:
Good point - I'll revise the wording of the motivation. I think this same plan can take us into GA.

Reply:
Great, my feeling is that we should define a plan without any reference to specific timepoints but with a comment to review if the volume of queries require us to reassess the policy

Reply:
I have re-worded this section a little further to remove the language about the Beta program.